prow/config.yaml

```diff
@@ -13738,6 +13738,33 @@ periodics:
       secret:
         secretName: velodrome-influxdb
 
+- name: periodic-features-no-freeze
+  interval: 1h
+  agent: kubernetes
+  spec:
+    containers:
+    - image: gcr.io/k8s-testimages/commenter:v20170808-abf66782
+      args:
```

Comment:
https://github.com/kubernetes/test-infra/blob/5ed9c588f9750706a8c84066e8c9e9c95975319f/robots/commenter/main.go#L63

Right now this will wait 3 hours after adding the frozen label to remove it `--updated=2h` and `interval: 1h`. Would we want to tighten this up a bit? 'd suggest `--updated=5m` `interval: 30m` or something